src/packet/BinaryWriter.js

```diff
@@ -1,137 +1,62 @@
-﻿'use strict';
+'use strict';
 /*
  * Simple BinaryWriter is a minimal tool to write binary stream with unpredictable size.
  * Useful for binary serialization.
  *
- * Copyright (c) 2016 Barbosik
- * 
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- *     http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 
- * 
+ * Copyright (c) 2016 Barbosik https://github.com/Barbosik
+ * License: Apache License, Version 2.0
  */
 
-function BinaryWriter(size) {
-    if (!size || size <= 0) {
-        size = Buffer.poolSize / 2;
-    }
-    this._buffer = new Buffer(size);
+function BinaryWriter() {
+    this._buffer = parseInt(process.version[1]) < 6 ? new Buffer(1e5) : Buffer.allocUnsafe(1e5);

```

Comment:
@Luka967 Spatial hashing is easy first off. Thats nothing unless it is a hierarchical spatial grid like I implemented.


Your c projects are useless. Other people have probably also made them too. And no one uses VBA.


Also, I finished the concept for CodersAssemble 4 months ago. 



And I dont care if you know a friend that can do it. You dont. You wasted your life.



And when did my rant point to OpenAgar? OpenAgar can anyway support it. 

![screen shot 2017-01-30 at 5 19 50 pm](https://cloud.githubusercontent.com/assets/13282284/22444296/59b99a9a-e710-11e6-8acc-206a4f619b0a.png)